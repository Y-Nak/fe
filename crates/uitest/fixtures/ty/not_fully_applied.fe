pub struct Foo<T, U> {
    t: T
    u: U
}

pub struct Bar {
    f: Foo<i32>
}

pub enum Baz {
    Variant(Foo<i32>)
}