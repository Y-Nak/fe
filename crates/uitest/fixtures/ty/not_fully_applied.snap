---
source: crates/uitest/tests/ty.rs
expression: diags
input_file: crates/uitest/fixtures/ty/not_fully_applied.fe
---
error[3-0001]: expected fully applied type
  ┌─ not_fully_applied.fe:7:8
  │
7 │     f: Foo<i32>
  │        ^^^^^^^^ expected fully applied type here

error[3-0002]: kind mismatch in type application
   ┌─ not_fully_applied.fe:11:13
   │
11 │     Variant(Foo<i32>)
   │             ^^^^^^^^ can't apply `(* -> (* -> (* -> *)))` kind to `(* -> *)` kind


